Seed.Api/Controllers/UserController.cs

```diff
@@ -58,18 +58,19 @@ public UserController(IUserService userService)
         /// Creates a new user
         /// </summary>
         /// <param name="user" cref="UserDto">User model</param>
-        /// <response code="204">User created</response>
+        /// <response code="201">User created</response>
         /// <response code="404">User could not be created</response>
```

Comment:
This should be 400: bad request. Do you mind updating it?

Reply:
you're right, but, the 400 it's auto defined when a ValidateModel attribute is decorating the method, so i only need to remove it c:

Reply:
Not sure to understand, why you say that 400 is not needed? On the other hand, if there is not 404 response we should delete that one.

Reply:
oh, yes, i forgot to push the change, sorry